README.md

```diff
@@ -22,6 +22,9 @@ Official binary sources (provided by davidgfnet):
 Building
 --------
 
+Dependencies:
+Libc6-dev, libfreeimage-dev
```

Comment:
A lot of linux users start with an ubuntu based distro, so as a result,
most "beginners" use ubuntu/debian. I think we should start getting a
little more beginner friendly if we want linux to be more popular, and not
ridicule them.
On Aug 13, 2015 1:16 PM, "David G. F." <notifications@github.com> wrote:

> In README.md
> <https://github.com/davidgfnet/whatsapp-purple/pull/304#discussion_r36963608>
> :
>
> > @@ -22,6 +22,9 @@ Official binary sources (provided by davidgfnet):
> >  Building
> >  --------
> >
> > +Dependencies:
> > +Libc6-dev, libfreeimage-dev
>
> Hahahaha do they? Are Debian users retarded? :P Yeah lots of Ubuntu folks,
> but come on, they do have a launchpad repo... Let them suffer a bit
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/davidgfnet/whatsapp-purple/pull/304/files#r36963608>.
>